comp/api/api/apiimpl/utils/observability.go

```diff
@@ -48,7 +49,14 @@ func logResponseHandler(serverName string, getLogFunc func(int) logFunc) mux.Mid
 			code := lrw.Status()
 			logFunc := getLogFunc(code)
 
-			logFunc(logFormat, serverName, r.Method, r.RequestURI, r.RemoteAddr, duration, code)
+			var path string
+			// can't use r.URL.Path because http.StripPrefix could have been used
+			if reqURL, err := url.ParseRequestURI(r.RequestURI); err == nil {
+				path = reqURL.Path
```

Comment:
What do you think of adding the host or host:port as part of the log line? I think it would be useful for debugging.

Reply:
I started writing that it would just be the the value from the config, but since we have several APIs and it's a generic handler, the values would indeed vary (and the servers could bind to different hostnames, so that could be interesting)
So yeah definitely, good idea !

Reply:
@GustavoCaso following your proposition I updated the log format to also include the host (as given in the URL, it can have either `host` or `host:port` format) 👍